### PR TITLE
#113

### PR DESCRIPTION
**Pull request by eg04lt3r**

### CODE REVIEW

**Comment by davvd** [comment]
@eg04lt3r I will find a reviewer for your pull requests shortly, thanks for contribution!

**Comment by davvd** [comment]
@carlosmiranda review this one, please

File: src/main/java/org/takes/rs/RsWithHeaders.java [hunk 1]
```diff
@@ -80,4 +79,14 @@ public InputStream body() throws IOException {
...
     }
 
+    /**
+     * Wrap response and header into RsWithHeader.
+     * @param res Response
+     * @param header Header
+     * @return RsWithHeader
+     */
+    private static RsWithHeader wrapHeader(final Response res,
+                                           final String header) {
```
Comment (carlosmiranda) [inline src/main/java/org/takes/rs/RsWithHeaders.java]: Too much indentation here, there should be eight spaces only.

File: src/main/java/org/takes/http/BkTimeable.java [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+import java.io.IOException;
+import java.net.Socket;
+import lombok.EqualsAndHashCode;
+
+/**
+ * Back decorator with maximum lifetime.
+ * <p>The class is immutable and thread-safe.
+ * @author Aleksey Kurochka (eg04lt3r@gmail.com)
+ * @version $Id$
```
Comment (carlosmiranda) [inline src/main/java/org/takes/http/BkTimeable.java]: Let's add `@since 0.13` javadoc tag.

File: src/main/java/org/takes/http/BkTimeable.java [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+        final Thread callerThread = Thread.currentThread();
+        final Thread monitor = new Thread(
+            new Runnable() {
+                @Override
+                public void run() {
+                    try {
+                        Thread.sleep(BkTimeable.this.latency);
+                        callerThread.interrupt();
+                    } catch (final InterruptedException ex) {
+                        Thread.currentThread().interrupt();
```
Comment (carlosmiranda) [inline src/main/java/org/takes/http/BkTimeable.java]: Why interrupt the thread again when it's already been interrupted?

**Comment by carlosmiranda** [comment]
@eg04lt3r A few comments above. Furthermore, please edit the title to be more descriptive and do not leave the description empty - make sure you link to the original issue and briefly describe your solution.

**Comment by eg04lt3r** [comment]
@carlosmiranda please, review last commit. I deleted wrong code from this pull request. Sorry.

**Comment by carlosmiranda** [comment]
@eg04lt3r Please, update the ticket title and description according to our [QA rules](http://www.teamed.io/qa.html).

**Comment by davvd** [comment]
@elenavolokhova please, let us know what do you think about this ticket, according to our [QA rules](http://www.teamed.io/qa.html)

**Comment by elenavolokhova** [comment]
@carlosmiranda 
Thank you for the work on quality of this ticket! As far as it wasn't merged there is no reason to fix it's description now. 
In the future try to be more persistent and don't complete the review until such issues are fixed. We are on the right way :)

**Comment by elenavolokhova** [comment]
@davvd Quality is acceptable here.

**Comment by carlosmiranda** [comment]
@elenavolokhova Your comments are appreciated. Unfortunately the PR was closed before all issues were able to be addressed, we will definitely be more fastidious about quality next time.

**Comment by davvd** [comment]
> @davvd Quality is acceptable here.

@elenavolokhova thanks for the review, we'll try better next time

**Comment by davvd** [comment]
@carlosmiranda I added 10 mins to @elenavolokhova (for QA review) in transaction 56402079... Much obliged! I have added **18 mins** to your account in payment "56402090", 24 hours and 30 mins spent... extra minutes for review comments (c=3)... added +18 to your rating, now it is equal to [+4534](http://www.netbout.com/b/35023?open=rating)

**Pull request closed without merging**